server/server.py

```diff
@@ -147,8 +164,15 @@ def ListPrimitives(self, request, context):
     def Hello(self, request, context):
         return self.msg.make_hello_response_message()
 
+    def add_fitted_runtime(self, solution_id, runtime):
```

Comment:
Seems like these methods and those in `Server` don't really fit here, but I'm guessing there really was no good place to move the dict and the associated methods (original design assumed DB storage of pipelines).  Its better than having the dict declared as a global, but I just wanted to verify the reasoning.

Reply:
I mainly wanted to move away from global for now. I originally wanted to drop it into the server, but the server is not linked at all to the task manager (as far as I can tell) whereas the servicer is. Once we pickle them, this is not an issue.